### ISSUE TITLE
Some questions about the confidence mentioned in the paper

### ISSUE THREAD

**Issue by msn199959** [WangyiNTU/Point-supervised-crowd-detection#7]
Confidence and Order-aware Refinement Scheme was mentioned in the Algorithm 1, posterior probabilities would update the prior probabilities if posterior probabilities is larger. But how to ensure that the confidence is gradually increased during the training process, confidence was not taken into account in the loss function?

**Comment by WangyiNTU** [comment]
@msn199959 Hi, in Section III.D and Ablation study (Behavior of the confidence and order-aware refinement scheme), they show this phenomenon. Confidence is inversely proportional to loss.
